src/containers/SearchProvider.js

```diff
@@ -0,0 +1,27 @@
+import { geojson } from '../data/geojson';
+
+class SearchProvider {
+  async search({ query }) {
+    let matches = geojson.features.filter(
+      feature =>
+        feature.geometry &&
+        feature.geometry.coordinates &&
+        feature.geometry.coordinates.length > 0 &&
+        feature.properties.english_place_name
+          .toLowerCase()
+          .indexOf(query.toLowerCase()) > -1
```

Comment:
Move `query.toLowerCase()` out of the `filter` predicate and add a `lowerCaseQuery` variable.